Content.Shared/Damage/DamageSpecifier.cs

```diff
@@ -15,7 +15,7 @@ namespace Content.Shared.Damage
     ///     The actual damage information is stored in <see cref="DamageDict"/>. This class provides
     ///     functions to apply resistance sets and supports basic math operations to modify this dictionary.
     /// </remarks>
-    [DataDefinition]
+    [DataDefinition, Serializable]
```

Comment:
This is specifically because of the stun baton and welder right now calling for a force toggle when it runs out of charge/fuel. Since the event is raised by server side systems, the client's entity melee damage doesn't get changed back. All the other fields seem to be networked.

Reply:
Done.